app/src/main/java/com/mensinator/app/settings/SettingsScreen.kt

```diff
@@ -126,6 +128,26 @@ fun SettingsScreen(
             },
             onOpenIntPicker = { viewModel.showIntPicker(it) }
         )
+        SettingText(
+            text = stringResource(StringSetting.PERIOD_NOTIFICATION_MESSAGE.stringResId),
+            onClick = { viewModel.showPeriodNotificationDialog(true) }
+        )
+
+        if (viewState.showPeriodNotificationDialog) {
+            val initMessage = viewState.periodNotificationMessage
+            val messageKey = ResourceMapper.getStringResourceId(initMessage)
+            val messageText = messageKey?.let { stringResource(id = it) } ?: initMessage
+            NotificationDialog(
+                messageText = messageText,
+                onSave = {
```

Comment:
specify the parameter name of this lambda (right now it's `it`)

Reply:
This was also taken from the symptom screen so maybe it should be changed there too

Reply:
Changed, not sure if you wanted me to specify in the `onSave` lambda as well

Reply:
Oh I meant onSave, sorry!

Reply:
no problem, changed it to how I think you want it now :)